### PR TITLE
DEVEX-2292 `--detailed-job-metrics` for `dx run`

### PR DESCRIPTION
**Pull request by kpjensen**

### CODE REVIEW

File: src/python/dxpy/scripts/dx.py [hunk 1]
```diff
@@ -5270,9 +5271,10 @@ def positive_integer(value):
...
-parser_run.add_argument('-r', '--rank', type=int, help=fill('Set the rank of the root execution, integer between -1024 and 1023. Requires executionRankEnabled license feature for the billTo. Default is 0.', width_adjustment=-24), default=None)
+parser_run.add_argument('-r', '--rank', type=int, default=None, help=fill('Set the rank of the root execution, integer between -1024 and 1023. Requires executionRankEnabled license feature for the billTo. Default is 0.', width_adjustment=-24))
 parser_run.add_argument('--max-tree-spot-wait-time', help=fill('The amount of time allocated to each path in the root execution\'s tree to wait for Spot (in seconds, or use suffix s, m, h, d, w, M, y)', width_adjustment=-24))
 parser_run.add_argument('--max-job-spot-wait-time', help=fill('The amount of time allocated to each job in the root execution\'s tree to wait for Spot (in seconds, or use suffix s, m, h, d, w, M, y)', width_adjustment=-24))
+parser_run.add_argument('--detailed-job-metrics', action='store_true', default=None, help=fill('Enabled detailed metrics for the execution. Requires detailedJobMetrics license feature for the billTo.', width_adjustment=-24))
```
Comment (wormsik) [inline src/python/dxpy/scripts/dx.py:5277]: is there any way how to turn off detailed metrics? let's assume org has policy default = `True` and someone wants to turn the collection off for one specific job...

Comment (wormsik) [inline src/python/dxpy/scripts/dx.py:5277]:
```suggestion
parser_run.add_argument('--detailed-job-metrics', action='store_true', default=None, help=fill('Enable detailed metrics for the execution. Requires detailedJobMetrics license feature for the billTo.', width_adjustment=-24))
```

**Review by wormsik — CHANGES_REQUESTED** [review]

File: src/python/dxpy/scripts/dx.py (hunk 1 above)
Comment (kpjensen) [reply src/python/dxpy/scripts/dx.py:5277]: It's possible in the `/run` API, but not via the CLI parameter.

Comment (wormsik) [reply src/python/dxpy/scripts/dx.py]: ok thx

**Review by wormsik — APPROVED** [review]
LGTM 👍

**Pull request merged**